mdnDocsOverrides.json

```
@@ -585,16 +659,24 @@
     "Web/CSS/display#animating_display",
     "Web/CSS/content-visibility#animating_and_transitioning_content-visibility"
   ],
-  "display-contents": [],
-  "display-flow-root": [],
+  "display-contents": [
+    "Web/CSS/display#contents"
+  ],
+  "display-flow-root": [
+    "Web/CSS/display#flow-root"
+  ],
   "display-list-item": [
     "Web/CSS/display-listitem"
   ],
   "display-mode": [
     "Web/CSS/@media/display-mode"
   ],
-  "display-ruby": [],
-  "display-table": [],
+  "display-ruby": [
+    "Web/CSS/display-internal"
```

I think it makes more sense to link to the relevant section of the main `display` property, where people can learn more about the various display values.

```suggestion
    "Web/CSS/display#ruby"
```